examples/bot.py

```diff
@@ -10,7 +10,10 @@
 # When you make a bot, we refer to it as the "client."
 # The client is the main object that interacts with the Gateway, what talks to Discord.
 # The client is also the main object that interacts with the API, what makes requests with Discord.
-client = interactions.Client()
+# The client can also have "intents" that are what the bot recieves,
+# in this case the default ones and message content
+intents = interactions.Intents.DEFAULT | interactions.Intents.MESSAGE_CONTENT
```

Comment:
Please add a note that message content is a special intent that requires toggling it on in the dashboard too.